src/components/Rockets/Rockets.css

```diff
@@ -0,0 +1,18 @@
+.rocket {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  margin-top: 1rem;
+  margin-left: 10rem;
+}
+
```

Comment:
[OPTIONAL] Good job styling your UI;
I suggest you add some styling to your reserve and cancel reservation button so that it looks more like the requirement.